data/101/788/295/101788295.geojson

```diff
@@ -144,7 +144,7 @@
     "wof:lang":[
         "hun"
     ],
-    "wof:lastmodified":1582345341,
+    "wof:lastmodified":1634678111,
     "wof:name":"\u00d3csa",
```

Comment:
Nit: Generally `wof:name` should be ASCII7 which wouldn't have diacritical marks.